### DISCUSSION TITLE
how to convert the Euler angles to rotation matrix

### DISCUSSION THREAD

**Discussion by ykzzyk** [pyvista/pyvista#3597] [Q&A]
Hello y'all, I've been trying to convert the Euler angles to a rotation matrix using Scipy. But the results are slightly different.

For example, I have: 
Euler angles [6.835578651406617, 47.91692755829381, 172.76787223914218] (in degrees), 
and translation vector [2.5987030981091648, 31.039133701224685, 14.477777915423951]. 

I converted them to a rotation matrix using Spicy, and I got:
array([[-0.66487539, -0.21262585, -0.71605235,  2.5987031 ],
       [ 0.08437209, -0.97387229,  0.21084143, 31.0391337 ],
       [-0.74217388,  0.07976845,  0.66544341, 14.47777792],
       [ 0.        ,  0.        ,  0.        ,  1.        ]])

But the results are slightly different when i use the Euler angle to perform transformation vs using the rotation matrix to do it.
Euler angle transformation: 
![euler](https://user-images.githubusercontent.com/55161270/201727176-62d88696-ef7d-4be5-a909-4c03694c88a2.png)

rotation matrix transformation:
![rotation_matrix](https://user-images.githubusercontent.com/55161270/201727240-ef55b3ca-b7dc-4f29-9ffe-21fc2e3f87d2.png)

**Comment by darikg** [comment]
Can you post a minimal example? Euler rotations depend on the order in which they're applied and whether they're intrinsic or extrinsic -- I assume that's the difference between the two methods
